Libraries/LibRegex/RegexParser.cpp

```diff
@@ -187,6 +187,10 @@ Parser::Result Parser::parse(Optional<AllOptions> regex_options)
     else
         set_error(Error::InvalidPattern);
 
+    Vector<FlyString> capture_groups;
+    for (auto const& [name, _] : m_parser_state.named_capture_groups)
+        capture_groups.append(name);
```

Comment:
This can be replaced with `auto capture_groups = m_parser_state.named_capture_groups.keys();`